tests/fsas_sir/show_ether/fsas_sir_show_ether.yml

```
@@ -0,0 +1,82 @@
+---
+parsed_sample:
+  - config_mdi: "mdi"
+    config_media: "-"
+    config_mode: "100M"
+    description: "__@@foo"
+    ether_group: "1"
+    ether_port: "1"
+    flow_control_receive: "on"
+    flow_control_send: "off"
+    media: "Metal"
+    since: "Feb 16 21:28:21 2024"
+    status: "100M Full MDI"
+    status_auto: ""
+    status_duplex: "Full"
+    status_mdi: "MDI"
+    status_speed: "100M"
```

It's a bit of a bummer the status data is captured twice.
I do notice that's likely because of the `offline (backup)` and `disable` statuses.

---

Thank you for the review. Would you prefer not to capture `STATUS` when not `offline` or `disalble`?
```suggestion
    status: ""
    status_auto: ""
    status_duplex: "Full"
    status_mdi: "MDI"
    status_speed: "100M"
```

```ntc_templates/templates/fsas_sir_show_ether.textfsm
  ^status\s+:\s+(${STATUS_AUTO}\s+)?${STATUS_SPEED}\s+${STATUS_DUPLEX}\s+${STATUS_MDI}\s*$$
```

---

> Thank you for the review. Would you prefer not to capture `STATUS` when not `offline` or `disalble`?
> 
> ```
>   ^status\s+:\s+(${STATUS_AUTO}\s+)?${STATUS_SPEED}\s+${STATUS_DUPLEX}\s+${STATUS_MDI}\s*$$
> ```

@jmcgill298 @jvanderaa @itdependsnetworks
What are your thoughts on the `STATUS` capturing data that _sometimes_ is captured in other capture groups?

The downside here is that the `STATUS` capture group would have to become much more rigid (to be the two statuses in the present test data) so it doesn't match the speed/duplex/MDI.

---

I think if the goal is to split the status field into multiple capture groups, then the status field should capture `auto`, `disable`, and `offline` text, and then it would be an empty string if it is set to a particular speed.

---

> I think if the goal is to split the status field into multiple capture groups, then the status field should capture `auto`, `disable`, and `offline` text, and then it would be an empty string if it is set to a particular speed.

I like this idea.
I cooked up some suggestions.